1.9/metrics/datadog.md

```diff
@@ -0,0 +1,135 @@
+---
+post_title: Datadog Metrics for DC/OS
+feature_maturity: preview
+menu_order: 3.5
+---
+
+The Datadog metrics plugin for DC/OS supports sending metrics from the DC/OS metrics service on master and agent hosts to a Datadog agent for shipping to DatadogHQ.
+
+# Installation
+
+## Build the plugin
+
+**Prerequisite:** 
+- DC/OS is installed
+- [Go programming environment](https://golang.org/doc/install) <!-- dcos-metrics must be run from within the go directory -->
+- GitHub
+
+1. Clone the DC/OS metrics repository:
+
+   ```bash
+   $ git clone git@github.com:dcos/dcos-metrics
+   ```
+   
+1. Navigate to the `dcos-metrics` repository and run the build command:
+
+   ```bash
+   $ cd dcos-metrics && make && make plugins
+   ```
+
+   The plugin is available in the build directory:
+
+   ```
+   $ tree build
+   build
+   ├── collector
+   │   └── dcos-metrics-collector-1.0.0-rc7
+   ├── plugins
+   │   └── dcos-metrics-datadog_plugin-1.0.0-rc7
+   └── statsd-emitter
+       └── dcos-metrics-statsd-emitter-1.0.0-rc7
+   ```
+
+## Install the Datadog agent in your DC/OS cluster
+
+Install the `datadog` package in DC/OS:
+
+1.  Go to the **Universe** tab of the DC/OS GUI and find the **Datadog** package. 
+    ![datadog package](/docs/1.9/img/datadog-package.png)
+1.  Click **INSTALL PACKAGE** -> **ADVANCED INSTALLATION** and enter [your Datadog API_KEY](https://app.datadoghq.com/account/settings#api).
+1.  Click **REVIEW AND INSTALL** to complete your installation.
+
+After a minute or two a Datadog agent will be running in the cluster at `datadog-agent.marathon.mesos:8125`. This is the default location used by the Datadog plugin.
+
+## Test the DC/OS Datadog metrics plugin (agents only)
+As a stopgap during testing, you may be able to manually run the Datadog plugin on your agents by running it as a Marathon task. You must first upload your binary to a web server that's visible to your cluster, then create a Marathon application like the following (with customized `cmd`, `instances`, and `uris` to meet your needs):
+
+```json
+{
+  "cmd": "chmod +x ./dcos-metrics-* && ./dcos-metrics-* -dcos-role agent -auth-token <CONTENT OF 'dcos config show core.dcos_acs_token'>",
+  "instances": NUMBER_OF_AGENTS,
+  "uris": [ "https://YOURFILEHOST.COM/dcos-metrics-datadog_plugin-YOURBUILDVERSION" ],
+  "id": "test-datadog-plugin",
+  "cpus": 0.1,
+  "mem": 128,
+  "disk": 0,
+  "acceptedResourceRoles": [ "slave_public", "*" ]
+}
+```
+
+### Install the DC/OS Datadog metrics plugin
+When you're happy with the test results, you'll need to install the plugin into your cluster. For each host in your cluster, transfer your binary for the plugin and then add a systemd unit to manage the service. This unit differs slightly between agent and master hosts.
```

Comment:
The only reason to do this step (unless you're a plugin developer) is to check that metrics are being piped to DataDog via the agent as you expect. So 'success' would just be data showing up in your dashboard.